docs/_index.yaml

```diff
@@ -4,25 +4,30 @@ description: <!--no description-->
 landing_page:
   custom_css_path: /site-assets/css/style.css
   rows:
-  - heading: TensorFlow's visualization toolkit
+  - heading: "TensorBoard: TensorFlow's visualization toolkit"
     items:
     - classname: devsite-landing-row-50
       description: >
         TensorBoard provides the visualization and tooling needed for 
-        machine learning experimentation. It enables:
+        machine learning experimentation:
         <ul style="padding-left: 20px; margin: 20px 0;">
           <li>Tracking and visualizing metrics such as loss and accuracy</li>
           <li>Visualizing the model graph (ops and layers)</li>
-          <li>Viewing histograms and distributions of Tensors (e.g., weights 
-          and biases) to track how they change over time</li>
+          <li>Viewing histograms of weights, biases, or other tensors as they
+              change over time</li>
           <li>Projecting embeddings to a lower dimensional space</li>
           <li>Displaying images, text, and audio data</li>
           <li>Profiling TensorFlow programs</li>
           <li>And much more</li>
         </ul>
-
       image_path: /tensorboard/images/tensorboard.gif
-
+      buttons:
+      - label: Learn More
```

Comment:
The "learn more" here is a bit confusing. I likely landed here by clicking on "learn more" from some other page. Maybe we explicitly refer to visiting the r1 and r2 docs?

Reply:
I guess that didn't seem like an issue to me - I think it would be a continuation of the "learning more" flow since it's still progressively more detail about TensorBoard, and this is basically just an interstitial in that case.  I was trying to avoid framing it just as "r1 and r2" docs since 1) I was trying to make the alpha docs sound more exciting to encourage users to go there for the new stuff and 2) it makes them sound like parallel experiences, even though the content they have is quite different.

I also wasn't sure that we wanted to specifically say just e.g. "View the Docs" in the call-to-action buttons since it sounds relatively dry.

Is there a specific wording you have in mind?

Reply:
I agree. Keeping it like this seems fine. 
Maybe another idea is to swap the order and have TF 2.0 alpha first to really emphasize that's the exciting stuff, but maybe that just adds to the confusion.

Reply:
Hmm, I tried that on staging but looking at it, I think that might add to the confusion since it's less clear then that "Learn More" is generally about TensorBoard versus about the action to explore the alpha?

It also seemed a little more consistent to keep the alpha on the right in both the navbar and the buttons, and kind in keeping with it being the "upcoming" thing.  My thinking was sort of that we have a standard "learn more" continuation action, but then include the alpha as a teaser button to pull the reader's eye off the beaten track towards the new and exciting thing.  Maybe I'm overthinking it though.

Reply:
Yeah, I think the current approach sounds like a great start.